meteor/client/ui/FloatingInspectors/NoraFloatingInspector.tsx

```diff
@@ -152,32 +152,50 @@ export class NoraPreviewRenderer extends React.Component<{}, IStateHeader> {
 		this.rootElement = e
 	}
 
-	private getElStyle() {
+	private getElStyle(width: number, height: number) {
 		const style = { ...this.state.style }
 		style.visibility = this.state.show ? 'visible' : 'hidden'
+		style.width = `${width / 4}px`
+		style.height = `${height / 4}px`
```

Comment:
I think we should make the viewed size be independent of the absolute rendered values.
So something like this instead?

```typescript
const ratio = height / width
const idealWidth = Math.max(300, browserWidth/4)
style.width = `${idealWidth}px`
style.height = `${idealWidth * ratio}px`

```

Reply:
I am open to this, but I think it needs a team/UX discussion to figure out what rule it should follow.

Reply:
After discussion with the team, we agreed on making the preview be a maximum of 480px in either dimension